src/olympia/amo/utils.py

```diff
@@ -881,8 +881,13 @@ def _open(self, name, mode='rb'):
                     raise
         return super()._open(name, mode=mode)
 
-    def path(self, name):
-        return os.path.normpath(super().path(force_str(name)))
+    def path(self, *names):
```

Comment:
Do we need this ? I'm not sure we want to change the signature of a method that is shared by all Django storage classes (https://docs.djangoproject.com/en/5.1/ref/files/storage/#django.core.files.storage.Storage.path) Also this is not tested (we have `src/olympia/amo/tests/test_storage_utils.py`).

Reply:
/agree - if you need this it should be a different(ly named) method

Reply:
Ah, looked in test_utils. It is a simple change and is fully backwards compatible. I'll add a test for multiple paths.

And to original question, yes we do.

Reply:
@eviljeff can you explain why it would **need** to be a differently named method? Nothing about the current use of the method would change, and you would not need to consider should I use "path" or "path_join".

Reply:
it's a standard interface to the Storage class, so we wouldn't be able to swap it out as easily.

Reply:
Fair enough. Removed. Added the test that was seemingly missing though for the original implementation.